tsconfig.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
Are you sure all these compiler settings are needed for this Vercel project? They probably have some default one somewhere and pretty sure it will be super short

Reply:
Yeah good point, probably not all needed. I'll have a look into

Reply:
I've swapped it to use the default that vercel creates, which isn't too different

Reply:
To me this looks like the default Next.js tsconfig, and again, this is not a Next.js app. Pretty sure we don’t need more than this, and we especially should not define any custom `outDir` https://github.com/oceanprotocol/dockerhub-proxy/blob/main/tsconfig.json

Reply:
Yep, I've simplified this